packages/create-catalyst/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "display": "Node",
   "compilerOptions": {
     "strict": true,
-    "target": "es6",
+    "target": "es2023",
```

Comment:
Why was this needed?

Reply:
`const last = items.findLastIndex(isSelectable);`

findLastIndex() was added since 2022 year and works in all latest browsers and NodeJS 18+ versions
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findLastIndex